setup.sh

```diff
@@ -12,7 +12,9 @@ if [ "${0}" != "${BASH_SOURCE}" ]; then
     echo "      ${STACK_PATH}"
     source ${STACK_PATH}
   else
-    source /cvmfs/sw.hsf.org/key4hep/setup.sh
```

Comment:
By default stable stack should be used. If you want to use nightlies than source the nightlies stack before you source fccanlayses setup script. Please revert this change.